docs/docs/theme-api.md

```diff
@@ -89,27 +89,27 @@ import Header from "gatsby-theme-amazing/src/components/header"
 export default props => <Header {...props} myProp="true" />
 ```
 
-Taking this approach means that when we upgrade our theme later we can also take advantage of all the updates to the `Header` component because we haven't fully replaced it, just modified it.
+Taking this approach means that when you upgrade your theme later you can also take advantage of all the updates to the `Header` component because you haven't fully replaced it, just modified it.
 
 ### What path should be used to shadow a file?
 
-Until we build tooling to support automatically handling shadowing, you will have to manually locate paths in a theme and create the correct shadowing paths in your site.
+Until gatsby has tooling to automatically handle shadowing, you will have to manually locate paths in a theme and create the correct shadowing paths in your site.
```

Comment:
```suggestion
Until Gatsby has tooling to automatically handle shadowing, you will have to manually locate paths in a theme and create the correct shadowing paths in your site.
```

I feel like we generally capitalize Gatsby when we mention it like this 🙂

Reply:
done 👍